BeerProject/src/main/java/wawer/kamil/beerproject/service/impl/BeerServiceImpl.java

```diff
@@ -147,6 +163,7 @@ public Beer updateBeerByBreweryIdAndBeerId(Long breweryId, Long beerId, Beer upd
     //delete beers
 
     @Override
+    @CacheEvict(value = "beerCache", beforeInvocation = false)
```

Comment:
redundantny parametr. Domyslnie jest false

Reply:
Zmieniłam to na allEntries = true, mamy do wyboru usunięcie z cashe jednego wpisu któy zostaje usuniety z bazy, ale przy tej operacji usunięcie wszystkich. Wybrałam wszystkie dlatego że to jest pamięc szybkiego dostepu i ona szybko się zapełnia więc chyba dobrze było by ją też czyścić przy okazji operacji delete. Jak myślisz?